app/services/projects/archive_service.rb

```diff
@@ -48,8 +48,9 @@ def persist(service_call)
     end
 
     def archive_project(project)
-      # we do not care for validations
-      project.update_column(:active, false)
+      project.active = false
+      # We do not care for validations but want the timestamps to be updated
+      project.save(validate: false)
```

Comment:
I would prefer to use `update_attribute` in this case, which is designed for this use case (saving without validations, but touching + executing callbacks)

```suggestion
      project.update_attribute(:active, false)
```

https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attribute

Reply:
I wasn't aware of the differences between `update_attribute` and `update_column`. Good to know. Thanks.